### ISSUE TITLE
make site can't find dists/categories/current

### ISSUE THREAD

**Issue by Stebalien** [ipfs/distributions#155]
For some reason, `make site` goes looking for `dists/categories/current` and can't find it. This blocks `make publish`

```
./deps-check.sh
yarn install v1.3.2
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 0.37s.
echo "** Building site **"
** Building site **
npm run build

> dist.ipfs.io@1.0.0 build /home/steb/projects/go/src/github.com/ipfs/distributions
> npm run dist && npm run build:assets && hugo -s site -d ../releases


> dist.ipfs.io@1.0.0 dist /home/steb/projects/go/src/github.com/ipfs/distributions
> node scripts/dist.js

{ Error: ENOENT: no such file or directory, open '/home/steb/projects/go/src/github.com/ipfs/distributions/dists/categories/current'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/steb/projects/go/src/github.com/ipfs/distributions/dists/categories/current' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dist.ipfs.io@1.0.0 dist: `node scripts/dist.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dist.ipfs.io@1.0.0 dist script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/steb/.npm/_logs/2017-11-05T20_47_04_235Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! dist.ipfs.io@1.0.0 build: `npm run dist && npm run build:assets && hugo -s site -d ../releases`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the dist.ipfs.io@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/steb/.npm/_logs/2017-11-05T20_47_04_249Z-debug.log
make: *** [Makefile:21: site] Error 1
```

hugo version 0.30.2

**Comment by kevina** [comment]
@Stebalien this is fixed in #160, correct?

**Comment by Stebalien** [comment]
Ah. Yes.
